app/packages/operators/src/SplitButton.tsx

```diff
@@ -63,6 +63,7 @@ export default function SplitButton({
   };
 
   const handleSelect = (option) => {
+    if (!option.onSelect) return;
```

Comment:
I noticed this bug while testing. Clicking on the Schedule button in the modal threw an error. This fixes it.